config.local.toml

```diff
@@ -0,0 +1,34 @@
+# Zola config for compiling examples locally
```

Comment:
IMO we shouldn't maintain a separate nearly identical copy of this file just for this purpose. I would move the local versions of `wasm_webgl2_base_url` to comments in `config.toml`.

Reply:
Personally, i'd love to not need either, but Zola is limited in this regard. Maybe comments can work, but my goal was that there won't be a need to edit a git tracked file for at least the basic usecases.

Reply:
Was just working with these changes myself to help troubleshoot another issue, and having the config predefined absolutely helped smooth out the process. 

I don't think it's particularly uncommon to have a local config file for this sort of thing, and it doesn't look like Zola supports extending configs. 

How often do these config files change? Will having an additional config file meaningfully increase maintenance costs?